py3status/modules/xrandr.py

```diff
@@ -138,13 +136,8 @@ def _get_layout(self):
             'disconnected': OrderedDict()
         })
 
-        current = Popen(['xrandr'], stdout=PIPE)
-        for line in current.stdout.readlines():
-            try:
-                # python3
-                line = line.decode()
-            except:
-                pass
+        current = self.py3.command_output('xrandr')
+        for line in current.splitlines(True):
```

Comment:
Existing bugs for me. I read up. `.splitlines(True)` to stay in consistent behavior as `readline()`.. to stay on safe side. Can omit if you want to.